src/flb_utils.c

```diff
@@ -801,6 +801,11 @@ int flb_utils_write_str(char *buf, int *off, size_t size, const char *str, size_
     off_t offset = 0;
     char tmp[16];
     char *p;
+#if defined(FLB_SIMD_RVV)
```

Comment:
callers should not be aware of what's being used, it should be agnostic

Reply:
Probably, we should define the length of this instruction as a macro instead of using to store into a variable?
This should be effective for the caller.
The reason of this ifdef is used for determining the instruction length.
This is because RISC-V's RVV which is vector instruction does not have a fixed length. So, we need to provide the fixed length.